tensorflow/lite/delegates/gpu/cl/kernels/BUILD

```diff
@@ -244,7 +244,7 @@ cc_test(
         "//tensorflow/lite/delegates/gpu/common:operations",
         "//tensorflow/lite/delegates/gpu/common:status",
         "//tensorflow/lite/delegates/gpu/common/tasks:convolution_transposed_thin_test_util",
-        "@com_google_googletest//:gtest_main_no_heapcheck",  # constant buffers leak on nvidia
```

Comment:
These changes have to stay due to a NVIDIA driver issue. Will be resolved internally once Nvidia updates.

Reply:
I'll remove this commit. It is stand-alone.

Am I using the wrong version of gtest? I get an error during building.